recipes/r-yyjsonr/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set version = '0.1.20' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-yyjsonr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/yyjsonr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/yyjsonr/yyjsonr_{{ version }}.tar.gz
+  sha256: ed46317a21b343ba613deef5a0c19a405d5c2fcc8814971e9d2b87968fa67b3e
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ stdlib('c') }}                # [not win]
+    - {{ stdlib('m2w64_c') }}          # [win]
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - libzlib
+  host:
```

Should be `zlib` in `host:`, i.e., try:

```suggestion
  host:
    - zlib
```

---

Thanks for the tip! I can never figure out when things need to be in build vs host...